Makefile

```diff
@@ -0,0 +1,86 @@
+# execute these tasks when `make` with no target is invoked
+default: unit-test reset e2e-test
+
+# declare non-file based targets to speed up target
+# invocataions by letting make know it can skip checking
+# for changes in a file
+.PHONY: lint install build publish unit-test e2e-test test up down restart reset refresh logs debug-postgres debug-cache
```

Comment:
These lists get difficult to maintain -- usually opt for defining the target as phony near the definition, though has no change in functionality.

Ex: make
```
.PHONY: lint
lint:
    ....

.PHONY: install
install:
    ...
```

Reply:
today I learned something new about `make`